guides/v2.1/extension-dev-guide/validate/test-module.md

```diff
@@ -1,58 +1,71 @@
+Test your component by deploying {{site.data.var.ce}} and adding the component to the project's `composer.json`. To install, see [Install using Composer].
 
-{% highlight JSON %}
+```json
 "require": {
     "magento/magento-composer-installer": "*",
-    "magento/product-community-edition": "2.0.0",
     "yourvendorname/module-one": "0.1.1"
 },
-{% endhighlight %}
+```
 
-Remember to [register]({{ page.baseurl }}/extension-dev-guide/build/component-registration.html) your component as well, adding the location of your component. Confirm that your component works as expected and Magento functionality is not compromised.
+Remember to [register] your component as well, adding the location of your component. Confirm that your component works as expected and Magento functionality is not compromised.
 
 ## Test installing your component {#test-install}
 
-Before you publish your component, you should test installing it using the [Magento Component Manager]({{ page.baseurl }}/comp-mgr/bk-compman-upgrade-guide.html){:target="_blank"} (part of the Magento Admin).
+Before you publish your component, you should test installing it using the [Magento Component Manager]. Access this feature through the part of the [Web Setup Wizard] in the Magento Admin.
 
 One way to do this follows:
 
-1.	[Package your component]({{ page.baseurl }}/extension-dev-guide/package/package_module.html) in a GitHub repository that's accessible by the machine on which you run the {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %}.
-2.	On that machine, create a static route from `https://repo.magento.com` to your GitHub repository.
+1.	[Package your component] in a GitHub repository that's accessible by the machine on which you run the {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %}.
+1.	On that machine, create a static route from `https://repo.magento.com` to your GitHub repository.
 
 	To create a static route, add a line similar to the following to your `hosts` file:
 
-		<IP or hostname of your GitHub repository> repo.magento.com
+		<IP or hostname of your GitHub repository> https://repo.magento.com
 
-3.	[Install your component]({{ page.baseurl }}/comp-mgr/module-man/compman-main-pg.html#compman-access){:target="_blank"} exactly like a merchant.
-4.	Verify the component installed properly.
+1.	[Install your component] exactly like a merchant.
+1.	Verify the component installed properly.
 
 ## More information
 
-For more information on testing in {% glossarytooltip bf703ab1-ca4b-48f9-b2b7-16a81fd46e02 %}PHP{% endglossarytooltip %} and validating Magento components, see the following:
-
-* [Validating Magento Extensions](http://www.gorillagroup.com/trending/insight/validating-magento-extensions-phpunit/){:target="_blank"}
+See these resources on testing in {% glossarytooltip bf703ab1-ca4b-48f9-b2b7-16a81fd46e02 %}PHP{% endglossarytooltip %} and validating Magento components:
```

Comment:
See the following resources for more information about testing in PHP and validating Magento components...